src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/default.phpt

```diff
@@ -0,0 +1,62 @@
+--TEST--
+Test DeprecationErrorHandler in default mode
+--SKIPIF--
+<?php if (!file_exists('DeprecationErrorHandler.php')) print "skip"; ?>
+--FILE--
+<?php
+
+putenv('SYMFONY_DEPRECATIONS_HELPER');
+
+define('PHPUNIT_COMPOSER_INSTALL', __DIR__.'/../../vendor/autoload.php');
```

Comment:
actually, we cannot use ``__DIR__`` in skipif as it is not replaced before evaluating the code (which is not done from a file anymore).